Sources/MapboxMaps/Snapshot/Snapshotter.swift

```diff
@@ -68,7 +69,13 @@ public class Snapshotter {
         self.mapSnapshotter = mapSnapshotter
         style = Style(with: mapSnapshotter)
         observable = mapboxObservableProvider(mapSnapshotter)
-        EventsManager.shared(withAccessToken: options.resourceOptions.accessToken).sendTurnstile()
+
+        sendTurnstileEvent(accessToken: options.resourceOptions.accessToken)
+    }
+
+    internal func sendTurnstileEvent(accessToken: String) {
+        let eventsManager = EventsManager(accessToken: accessToken)
```

Comment:
Seems strange that snapshotted doesn't hold onto the events manager - will the turnstile event be sent after events manager is deallocated?

Reply:
Yes, it is. EventsManager calls `flush` on its deallocation and Common implementation handles that case correctly.
Sources:
1. @tatiana-yan 
2. Checked traffic from Snapshotter example – there are two turnstiles. First for the MapView, and the second for Snapshotter